lib/Wikia/src/Util/Consul/ConsulConfig.php

```diff
@@ -0,0 +1,85 @@
+<?php
+namespace Wikia\Util\Consul;
+
+class ConsulConfig {
+
+	protected $urlSchema = "{tag}.{service}.service.{dataCenter}.consul";
```

Comment:
In current setup i believe we should keep "tag" required, because in other case we get services with "production" tag and "testing" tag.